package.json

```diff
@@ -19,6 +19,9 @@
     "build:cjs": "rimraf ./build && tsc --outdir build",
     "build:esm": "rimraf ./build/esm && tsc --outDir build/esm --module es2015",
     "build:copy-files": "node ./scripts/copy-files.js",
+    "lint": "eslint --fix 'src/**/*.{js,jsx,ts,tsx}'",
+    "lint-staged": "yarn lint-js-staged",
+    "lint-js-staged": "git diff --diff-filter=d --cached --name-only -z -- '*.js' '*.jsx' | xargs -0 -I % sh -c 'git show \":%\" | eslint --stdin --stdin-filename \"%\";'",
```

Comment:
There are no githooks, so we could remove these two. Or is travis executing them?